src/components/v2v3/shared/NftVideo/JuiceVideoThumbnailOrImage.tsx

```diff
@@ -11,50 +12,67 @@ export function JuiceVideoThumbnailOrImage({
   playIconPosition,
   heightClass,
   widthClass,
+  showPreviewOnClick,
   ...props
 }: ImageProps & {
   isSelected?: boolean
   playIconPosition?: PlayIconPosition
   heightClass?: string
   widthClass?: string
   src: string
```

Comment:
bump on this. isn't src a prop is ImageProps already?

Reply:
Yep but ImageProps allows it to be undefined which I didn't think was necessary.